tests/test_paperqa.py

```diff
@@ -662,7 +664,7 @@ async def embed_documents(self, texts):
     with subtests.test(msg="copying-after-get-evidence"):
         # After getting evidence, a shallow copy of Docs is not the same because its
         # texts index gets lazily populated, while a deep copy should preserve it
-        docs.get_evidence(
+        _ = await docs.aget_evidence(  # Changed from get_evidence to aget_evidence
```

Comment:
```suggestion
        _ = await docs.aget_evidence(
```

No need for this comment